src/styles/vars.scss

```diff
@@ -1,7 +1,10 @@
 // Sizes
 $navbar-width: 300px;
-$page-width: calc(100% - $navbar-width);
+$navbar-padding-sides: 2rem;
 $page-padding: 50px;
+$page-width: calc(
+    100% - $navbar-width - ($page-padding * 2) - $navbar-padding-sides
```

Comment:
If the navbar has a fixed `width` & `max-width` of 300px, why should the inside padding affect the page width? Might want to investigate the root cause.